includes/wc-deprecated-functions.php

```diff
@@ -867,3 +867,18 @@ function _wc_save_product_price( $product_id, $regular_price, $sale_price = '',
 		update_post_meta( $product_id, '_sale_price_dates_to', '' );
 	}
 }
+
+/**
+ * Get a log file path.
+ *
+ * @since 2.2
+ * @deprecated 2.7.0
+ *
+ * @param string $handle name.
+ * @return string the log file path.
+ */
+function wc_get_log_file_path( $handle ) {
```

Comment:
`WC_Log_Handler_File` is heavily dependent on this functionality, whereas `wc_get_log_file_path` outside of the context of the file log handler doesn't seem relevant.

That was the motivation for moving it to the class.